src/3d.js

```diff
@@ -33,6 +33,74 @@ The hackOptions below have relatively thorough documentation,
 but make sure to check out https://github.com/seleb/bitsy-hacks/wiki/3D-Hack-Examples 
 for examples of different setups if you're new to 3D.
 
+additional features to help make more fancy 3d scenes from bitsy editor:
+* make arcs, bridges, ceilings, multiple floors, and such by layering
+bitsy rooms on top of each other. add #tags to the names of the rooms
+you want to render together
+for example you can have rooms named 'theater #a0', 'stage #a1' and
+'catwalks #a4' and they will be displayd together, the 'stage'
+right on top of the 'theater', and 'catwalks' three tiles higher than the
+'stage'
+tags should be in this format: '#fjk12' where '#' comes first,
```

Comment:
yeah i think you are right. function format would be much cleaner and better to work with both from the code side and for folks to figure out how to use the hack because everything would be more consistent... the main reason i wanted to make tags as short as possible is because text areas for room/drawing names in bitsy editor are just so small and it's hard to work with tags if you can't see all parts of it at the same time.. but #stack(a,0) would fit just right into the room name area and it seems only a portion of rooms would use negative numbers or more than one digit or floats or long stack names anyway... it would be more complicated with drawings that tend to have several tags, but even in the 3d bitsy i'm working on currently they tend to be clutterd with the this old style tags just as well.. i guess it's toatlly worth it to adopt a new tag-function format for the sake of clarity!!! maybe it would also make it easy to print exceptions that would be helpful in case of invalid input instead of just ignoring it... i tried making drawing name area bigger with page inspection tools but i'm too bad at css haha i can't figure out what exactly makes them half the size they could have been

Reply:
RE making name inputs bigger: text inputs have a size defined in the HTML, but you can inject a stylesheet from the console to override it with some CSS:
```js
var s = document.createElement('style');
s.textContent = '.nameTextbox { width: 100%; }';
document.head.appendChild(s);
```

Reply:
oh wow! that worked just right. other things got messed up a little bit, but it's such a nifty workaround when you need to edit long drawing names, and then you can just reload the page and go back to normal